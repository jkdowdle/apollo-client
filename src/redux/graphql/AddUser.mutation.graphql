mutation AddUser($input: UserInput!) {
  addUser(input: $input) {
    id
    firstName
    age
  }
}