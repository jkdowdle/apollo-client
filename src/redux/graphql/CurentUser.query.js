import gql from 'graphql-tag'

export const CurrentUser = gql`
  query CurrentUser {
    currentUser {
      id
      email
    }
  }
`

export default CurrentUser