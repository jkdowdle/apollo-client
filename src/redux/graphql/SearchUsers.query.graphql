query SearchUsersQuery($input: SearchUser!) {
  searchUsers(input: $input) {
    id
    firstName
    age
  }
}