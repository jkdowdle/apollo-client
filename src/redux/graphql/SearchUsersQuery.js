import gql from 'graphql-tag'

export const SearchUsersQuery = gql`
  query SearchUsersQuery($input: SearchUser!) {
    searchUsers(input: $input) {
      id
      firstName
      age
    }
  }
`

export default SearchUsersQuery