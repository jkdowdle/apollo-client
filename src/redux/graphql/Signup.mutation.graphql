mutation Signup($email: String!, $password: String!) {
  signup(email: $email, password: $password) {
    id
    email
    jwt
  }
}