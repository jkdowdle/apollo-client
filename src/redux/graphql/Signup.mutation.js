import gql from 'graphql-tag'

export const Signup = gql`
  mutation Signup($email: String!, $password: String!) {
    signup(email: $email, password: $password) {
      id
      email
      jwt
    }
  }
`

export default Signup