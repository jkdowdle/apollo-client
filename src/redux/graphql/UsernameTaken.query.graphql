query UsernameTaken($value: String) {
  usernameAvailable(value: $value)
}