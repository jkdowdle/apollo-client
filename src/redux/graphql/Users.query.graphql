query UsersQuery {
  users {
    id
    firstName
    age  
  }
}