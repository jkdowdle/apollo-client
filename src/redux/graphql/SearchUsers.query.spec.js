// import { print } from 'graphql-tag/printer'
// import { print } from "graphql-tag/printer";
// import { bundledPrinter } from 'graphql-tag/printer'

import { print }from 'graphql/language'



import SEACH_USERS_QUERY from './SearchUsers.query.graphql'

describe("SEACH_USERS_QUERY", () => {
  it('should match the snapshot', () => {
    expect(print(SEACH_USERS_QUERY)).toMatchSnapshot()
  })
})