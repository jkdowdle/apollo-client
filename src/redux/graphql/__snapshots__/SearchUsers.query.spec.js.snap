// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEACH_USERS_QUERY should match the snapshot 1`] = `
"query SearchUsersQuery($input: SearchUser!) {
  searchUsers(input: $input) {
    id
    firstName
    age
  }
}
"
`;
